Task Liverpool/src/test/java/com/encora/automation/test/HomeTest.java

```diff
@@ -1,12 +1,33 @@
 package com.encora.automation.test;
 
+import org.testng.Assert;
+import pages.HomePage;
+import pages.ProductPage;
 import org.testng.annotations.Test;
 
 public class HomeTest extends BaseTest{
 
+    private HomePage homePage;
+    private ProductPage productPage;
+
+    private HomeTest(){
+        homePage = new HomePage(getDriver());
+        productPage = new ProductPage(getDriver());
+    }
+
     @Test
     private void TC1_SearchWithCode (){
-        System.out.println("test");
+        homePage.clickSearchBar();
+        homePage.typeSearchBar();
+        homePage.clickSearchBarButton();
+        Assert.assertEquals(productPage.getProductInfo().getProductName() , "Apple MacBook Pro (13 Pulgadas, Chip M1, 8 GB RAM, 256 GB SSD) - Gris Espacial", "Product Name is correct.");
```

Comment:
The message will be displayed when the assert fails, so it should be "Product name is incorrect"

The same for the other asserts

Reply:
fixed.